toggle-btn.vue

```diff
@@ -71,24 +70,36 @@ export default {
 	},
 	methods: {
 		setConfigData() {
-			if (this.options) {
-				if (this.options.handle) {
+			if (!this.options) {
+				return;
+			}
+
+			if (this.options.handle) {
+				[
+					PROP_KEYS.COLOR,
+					PROP_KEYS.DIAMETER,
+					PROP_KEYS.BORDER_RADIOUS
+				].forEach(element => {
+					this.setBindedProp('handle', element);
+				});
+			}
 
-					let propsToBind = [PROP_KEYS.COLOR, PROP_KEYS.DIAMETER, PROP_KEYS.BORDER_RADIOUS];
-					propsToBind.forEach(element => {
-						this.setBindedProp('handle', element);
-					});
-				}
-				if (this.options.track) {
-					// TODO - track border width, track border radius - 24.12.18
-					let propsToBind = [PROP_KEYS.COLOR, PROP_KEYS.WIDTH, PROP_KEYS.HEIGHT, PROP_KEYS.ACTIVE_COLOR, PROP_KEYS.BORDER_WIDTH, PROP_KEYS.BORDER_RADIOUS];
-					propsToBind.forEach(element => {
-						this.setBindedProp('track', element);
-					});
-				}
-				if (this.options.isActive) {
-					this.isActive = this.options.isActive;
-				}
+			if (this.options.track) {
+				// TODO - track border width, track border radius - 24.12.18
+				[
```

Comment:
i still think that it's more readable and user friendly to decalre what are this props used for.. 
your suggestion will work fine as well, but it depands how readable tou want to make it. 

it's kind of readable to see that you have some "props you want to bind" and then you iterate it, instaed of runnign on some array with consts and sending each const to a function

Reply:
For readability it can be extracted to a new (private) function, which is actually makes more sense